build.gradle

```diff
@@ -138,18 +138,24 @@ project(':iceberg-data') {
 }
 
 project(':iceberg-hive') {
-    dependencies {
```

Comment:
nit: Are the white space changes intentional? if not can you remove them?

Reply:
No need to revert, hard to tell if it was intentional or not so just confirming.